jobs/delete-version.groovy

```diff
@@ -28,9 +28,6 @@ Example: {"serviceA": ["version1", "version2"], "serviceB": ["version3"]}""",
 
 ).apply();
 
-currentBuild.displayName = "${currentBuild.displayName} (${params.VERSION})";
```

Comment:
How would it look to change this to:
```
currentBuild.displayName = "${currentBuild.displayName} (${params.SERVICE_VERSIONS})";
```
?

I know it's really wordy, but I wonder if it's unusably wordy.  I do say that the old code (that showed the version) was useful to me from time to time to figure out what has been reaped and what hasn't.